src/renderer/components/ft-profile-selector/ft-profile-selector.js

```diff
@@ -38,12 +38,25 @@ export default Vue.extend({
   mounted: function () {
     $('#profileList').focusout(() => {
       $('#profileList')[0].style.display = 'none'
+      // When pressing the profile button
+      // It will make the menu reappear if we set `profileListShown` immediately
+      setTimeout(() => {
+        this.profileListShown = false
+      }, 100)
     })
   },
   methods: {
     toggleProfileList: function () {
-      $('#profileList')[0].style.display = 'inline'
-      $('#profileList').focus()
+      const $profileList = $('#profileList')
```

Comment:
I would advise removing the dollar sign from the variable name out of consistency with the code base. This notation is not used for any other JQuery variables in the repo.